src/components/testCaseConfiguration/testCaseData/TestCaseData.tsx

```diff
@@ -92,11 +92,7 @@ const TestCaseData = () => {
           );
         })
         .catch((err) => {
-          handleToast(
-            "danger",
-            `Test Case dates could not be shifted. Please try again.`,
-            true
-          );
+          handleToast("danger", err?.response?.data?.message, true);
```

Comment:
Recommend having a default message in case `data.message` is not available.

Reply:
@jkotanchik-SB added default message.